app/api-v1/routes/attachment.js

```diff
@@ -1,11 +1,25 @@
-// eslint-disable-next-line no-unused-vars
+const logger = require('../../logger')
+const { errorResponse, FileUploadError } = require('../../utils/errors')
+
 module.exports = function (attachmentService) {
   const doc = {
     GET: async function (req, res) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      logger.info('Attachment upload: ', req.file)
+
+      try {
+        if (!req.file) {
+          throw new FileUploadError({ code: 400, message: 'No file uploaded', service: 'attachment' })
+        }
+
+        const attachment = await attachmentService.createAttachment(req.file)
+        const result = attachment[0]
+        res.status(201).json({ ...result, size: req.file.size })
+      } catch (err) {
+        errorResponse(res, err)
```

Comment:
I'd be tempted to create a wrapper function for handlers instead. That way we can abstract out the try catch as well